app/controllers/claims/support/schools_controller.rb

```diff
@@ -24,7 +24,9 @@ def school_options
 
   def check_school_option
     if school_form(javascript_disabled: true).valid?
-      redirect_to check_claims_support_schools_path(school: { search_urn: urn_param })
+      redirect_to check_claims_support_schools_path(
+        school: { id: school_params.fetch(:id), name: search_param },
```

Comment:
We need to trick the `javascript_disabled?` method here to get to the `check` method